dts/bindings/mtd/nxp,s32-qspi-hyperflash.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  QSPI hyperflash connected to the NXP S32 QSPI bus.
+
+compatible: "nxp,s32-qspi-hyperflash"
+
+include: "nxp,s32-qspi-device.yaml"
+
+properties:
+  device-id-word-addr:
+    required: true
+    type: int
+    description: |
+      The word address of the device ID in ASO (Application-Specific Object).
+      This address specifies the exact location within the memory where the device ID is stored.
+
+  rwds-low-dual-error:
+    type: boolean
+    description: |
+      Enable Read-Write Data Strobe (RWDS) dual error detect.
+
+  secure-region-locked:
+    type: boolean
+    description: |
+      The secure region is locked and cannot be accessed or modified.
+      This is particularly useful in scenarios where sensitive data needs protection from
+      unauthorized access, such as in financial applications or secure communication systems.
+      If it is disable, having access to all memory regions is beneficial during development
+      or debugging phases.
+
+  read-latency:
```

Comment:
having a second look at this, it seems simpler `read-latency-cycles = <x>` and use an int enum

Reply:
I updated

Reply:
suffix `-cycles`

Reply:
I updated